docs/reference/commandline/update.md

```diff
@@ -51,6 +51,10 @@ options on a running or a stopped container. On kernel version older than
 4.6, you can only update `--kernel-memory` on a stopped container or on
 a running container with kernel memory initialized.
 
+> **Warning**: `docker update` commands are not supported for Windows
```

Comment:
Should probably be singular here because there's only one update command here (unless you're counting `docker container update` and `docker update` separate)